db-service/lib/cqn2sql.js

```diff
@@ -6,16 +6,9 @@ const _strict_booleans = _simple_queries < 2
 
 const { Readable } = require('stream')
 
-const DEBUG = (() => {
-  const LOG = cds.log('sql-json')
-  if (LOG._debug) return cds.debug('sql-json')
-  return cds.debug('sql|sqlite')
-  //if (DEBUG) {
-  //  return DEBUG
-  // (sql, ...more) => DEBUG (sql.replace(/(?:SELECT[\n\r\s]+(json_group_array\()?[\n\r\s]*json_insert\((\n|\r|.)*?\)[\n\r\s]*\)?[\n\r\s]+as[\n\r\s]+_json_[\n\r\s]+FROM[\n\r\s]*\(|\)[\n\r\s]*(\)[\n\r\s]+AS )|\)$)/gim,(a,b,c,d) => d || ''), ...more)
-  // FIXME: looses closing ) on INSERT queries
-  //}
-})()
+const DEBUG = cds.debug('sql|sqlite')
+const LOG_SQL = cds.log('sql')
+const LOG_SQLITE =  cds.log('sqlite')
```

Comment:
Why do we actually explicitly check for sqlite here? Shouldn't sql only be good enough as it is DB agnostic?